proto/meta.proto

```diff
@@ -211,7 +211,7 @@ message DeleteWorkerNodeResponse {
 
 // Mark CN as schedulable or as unschedulable
 message UpdateWorkerNodeSchedulabilityRequest {
-  common.HostAddress host = 1;
+  uint32 worker_id = 1;
```

Comment:
IMHO it is not so bad if we break backwards compatibility, since I introduced `UpdateWorkerNodeSchedulabilityRequest` just today and we are not yet using it.